src/ios/ListLibraryItems.m

```diff
@@ -91,7 +96,6 @@ - (void)requestReadAuthorization:(CDVInvokedUrlCommand *)command {
     }
 }
 
-
```

Comment:
Not a big deal, but these changes (removing random empty lines) shouldn't be in the PR ;)